test/rulesets/Base/fastmath_able.jl

```diff
@@ -59,11 +59,11 @@ const FASTABLE_AST = quote
         end
         @testset "Multivariate" begin
             @testset "sincos(x::$T)" for T in (Float64, ComplexF64)
-                x, Δx, x̄ = randn(T, 3)
+                x = randn(T)
```

Comment:
is there any reason to keep this as a seperate variable and not just put `randn()` below?